front/src/routes/integration/all/nextcloud-talk/actions.js

```diff
@@ -0,0 +1,118 @@
+import { NextcloudTalkStatus } from '../../../../utils/consts';
+
+const actions = store => ({
+  updateNextcloudUrl(state, e) {
+    store.setState({
+      nextcloudUrl: e.target.value
+    });
+  },
+  updateNextcloudBotUsername(state, e) {
+    store.setState({
+      nextcloudBotUsername: e.target.value
+    });
+  },
+  updateNextcloudBotPassword(state, e) {
+    store.setState({
+      nextcloudBotPassword: e.target.value
+    });
+  },
+  updateNextcloudBotToken(state, e) {
+    store.setState({
+      nextcloudBotToken: e.target.value
+    });
+  },
+  async getNextcloudTalkSetting(state) {
+    store.setState({
+      nextcloudTalkGetSettingsStatus: NextcloudTalkStatus.Getting
+    });
+
+    let nextcloudUrl = '';
+    let nextcloudBotUsername = '';
+    let nextcloudBotPassword = '';
+    let nextcloudBotToken = '';
+
+    store.setState({
```

Comment:
Please call `store.setState` once, merging `nextcloudTalkGetSettingsStatus` and `nextcloud` vars